partiql-ast/src/main/java/org/partiql/ast/ddl/TableConstraint.java

```diff
@@ -13,7 +14,8 @@
 
 public abstract class TableConstraint extends AstNode {
     @Nullable
-    public final IdentifierChain name;
+    @Getter
+    protected final IdentifierChain name;
```

Comment:
If you want to truly hide this field, you could just handwrite these getters. Same with PathStep.

Reply:
Made the field private (same w/ `PathStep`). Lombok getter seems to work fine, so I just used that for now.